docker/docker-compose.yml

```diff
@@ -51,5 +61,16 @@ services:
       KAFKA_CREATE_TOPICS: >-
         inventory.instance:4:1
 
+  db:
+    container_name: db
+    image: postgres:13.4-alpine
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: okapi_modules
+
 volumes:
-  es-data: { }
```

Comment:
Is it ok not to have newline here?

Reply:
this line is removed, so yes
after adding persistent volume for PostgreSQL  `docker-compose` file looks good